### PR TITLE
Time lag stats

### PR DESCRIPTION
**Pull request by JRKagumba**
I extended cadence_features.py, with two new functions: compute_time_lag_stats and compute_time_lag_skew_kurtosis. The compute_time_lag_stats function calculates the mean and standard deviation of the time lags, while the compute_time_lag_skew_kurtosis function computes the skewness and kurtosis. The additions offer support for time lag distributions, offering insights into their central tendency, dispersion, and shape.

### CODE REVIEW

File: cesium/features/cadence_features.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "find_sorted_peaks",
     "peak_ratio",
     "peak_bin",
+    "compute_time_lag_stats",
+    "compute_time_lag_skew_kurtosis"
```
Comment (profjsb) [inline cesium/features/cadence_features.py:14]:
```suggestion
    "compute_time_lag_skew_kurtosis",
```

File: cesium/features/graphs.py [hunk 2]
```diff
@@ -144,6 +146,8 @@
         "stetson_j",
         "stetson_k",
         "weighted_average",
+        "time_lag_stats",
+        "time_lag_skew_kurtosis"
```
Comment (profjsb) [inline cesium/features/graphs.py:150]:
```suggestion
        "time_lag_skew_kurtosis",
```

**Review by profjsb — CHANGES_REQUESTED** [review]
Looking good. Thank you for your first PR here! Could you add a short test for the new features (in `cesium/features/test/test_cadence_features.py`)?

**Comment by JRKagumba** [comment]
Added test functions to validate the correctness of compute_time_lag_stats and compute_time_lag_skew_kurtosis by computing the mean, standard deviation, skewness, and kurtosis for a given array of time lags.

**Review by profjsb — APPROVED** [review]
@JRKagumba thanks again for your contribution. It turns out we have two of the four new features in the PR already.  Just so you know the four features are now called: "cads_avg", "cads_std", "cads_skew", and "cads_kurtosis".

To simplify the addition of the other two features I made use of `skew` and `kurtosis` helper functions in `common_functions.py`. The tests for these cadence statistics features looks like:

```python
def test_compute_time_lag_stats():
    """Test compute_time_lag_stats function."""
    times, values, errors = irregular_random(500)
    cads = np.diff(times)

    f = generate_features(
        times, values, errors, ["cads_avg", "cads_std", "cads_skew", "cads_kurtosis"]
    )
    npt.assert_almost_equal(f["cads_avg"], np.mean(cads))
    npt.assert_almost_equal(f["cads_std"], np.std(cads))
    npt.assert_almost_equal(f["cads_skew"], stats.skew(cads))
    npt.assert_almost_equal(f["cads_kurtosis"], stats.kurtosis(cads))
```

**Pull request merged**